test/library/standard/DataFrames/diten/readHDF5Df.good

```diff
@@ -0,0 +1,16 @@
+   s2_1   s1_1   s2_2   
+1   0.0      1    0.0   
+2   0.5      2    0.5   
+3  0.666667      3   0.666667   
+4  0.75      4   0.75   
+5   0.8      5    0.8   
+6  0.833333      6   0.833333   
+7  0.857143      7   0.857143   
+8  0.875      8   0.875   
+9  0.888889      9   0.888889   
+10  0.9     10    0.9   
+11 0.909091     11   0.909091   
+12 0.916667     12   0.916667   
+13 0.923077     13   0.923077   
+14 0.928571     14   0.928571   
+15 0.933333     15   0.933333   
```

Comment:
Was it intentional that the "real" files have identical content?

Reply:
Yes, it was intentional.  Since they're just examples with no real meaning it's fine for them to have the same data.